Soccar [UI]/src/nl/socnet/message/SwitchTeamMessage.java

```diff
@@ -13,21 +13,21 @@
 @MessageEvent(id = MessageConstants.SWITCH_TEAM_MESSAGE_ID, handler = SwitchTeamMessageHandler.class)
 public final class SwitchTeamMessage extends Message {
 
-    private final String username;
+    private final int playerId;
     private final TeamColour team;
 
-    public SwitchTeamMessage(String username, TeamColour team) {
-        this.username = username;
+    public SwitchTeamMessage(int playerId, TeamColour team) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)